docs/source/developer/build/BUILD_ANDROID.md

```diff
@@ -1,8 +1,8 @@
 # Build Android
 
-*Last Updated on December 21, 2019*
+*Last Updated on December 15, 2020*
```

Comment:
Is this date correct?

Reply:
Yeppers, it's all imported from the main repo so if it's incorrect then it should be fixed there.